install/kubernetes/helm/istio/charts/security/templates/enable-mesh-mtls.yaml

```diff
@@ -17,6 +17,7 @@ spec:
   peers:
   - mtls: {}
 ---
+{{- if not .Values.global.mtls.auto }}
```

Comment:
Actually I don't think we need this anymore. https://github.com/istio/istio/pull/16657 applies ISTIO_MUTUAL as long as authN is STRICT mtls and DR is not defined, regardless of mtls.auto. Let's do this in separate PR to make it easier to track.